components/scream/src/share/util/standalone_helper_functions.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef SCREAM_STANDALONE_TEST_UTILS_HPP
+#define SCREAM_STANDALONE_TEST_UTILS_HPP
+
+#include "share/field/field_manager.hpp"
+#include "share/grid/mesh_free_grids_manager.hpp"
+
+namespace scream {
+
+// Helper function to check the total water mass
+Real calculate_water_mass(const std::shared_ptr<const GridsManager>& grids_mgr,const std::shared_ptr<FieldManager>& field_mgr, const Real dt, const bool use_precip) {
```

Comment:
Here I suggest checking per column rather than over all columns that happen to be on the process, since conservation should hold in each column.